venv/lib/python3.9/site-packages/pip/_internal/network/session.py

```
@@ -0,0 +1,449 @@
+    elif data["implementation"]["name"] == 'IronPython':
+        # Complete Guess
+        data["implementation"]["version"] = platform.python_version()
+
+    if sys.platform.startswith("linux"):
+        from pip._vendor import distro
+
+        # https://github.com/nir0s/distro/pull/269
+        linux_distribution = distro.linux_distribution()  # type: ignore
+        distro_infos = dict(filter(
+            lambda x: x[1],
+            zip(["name", "version", "id"], linux_distribution),
```

Ruff (0.14.8)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)